nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/NviService.java

```diff
@@ -113,15 +105,31 @@ private static List<ApprovalStatus> generatePendingApprovalStatuses(Set<URI> ins
                    .toList();
     }
 
-    private static Candidate toPendingCandidate(CandidateDetails candidateDetails,
+    @JacocoGenerated
+    @SuppressWarnings("PMD.UnusedPrivateMethod")
+    private boolean exists(UUID uuid) {
```

Comment:
Not sure if I like this method. 
Why is it present while unused?
I would think  that this method is cheaper in execution than retrieving the entry, so first to a check if the entry exist, and then do the actual query.
It might become tempting to do multiple queries to the database retrieving the same data over again.